Assets/components/header/TheHeader.vue

```diff
@@ -32,11 +32,6 @@
                     class="header-login__username"
                 >
                     {{ loggedInUser.osu.username }} 
-                    <span 
-                        class="header-login__line" 
-                        :class="`header-login__line--${site}`"
-                    >|</span>
-                    {{ loggedInUser.discord.username }}
```

Comment:
re-add this back and remove the discriminator text in Server/passportFunctions.ts and Models/User.ts

Reply:
added back